main/install/configuration.dist.php

```diff
@@ -2156,6 +2156,11 @@
 // User extra fields to be check on user edition to generate a specific process if it was modified
 //$_configuration['user_edition_extra_field_to_check'] = 'ExtrafieldLabel';
 
+// Enable skills in subcategory to work independant on assignement
+// Require DB changes:
+// ALTER TABLE gradebook_category ADD allow_skills_by_subcategory tinyint(1) NOT NULL DEFAULT '1';
```

Comment:
Add the property to the GradebookCategory entity but disabled.

Reply:
agregado en el ultimo commit